packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/readme.md

```diff
@@ -1,6 +1,6 @@
 <% if (projectDescription) { %># <%=projectDescription%><% } %>
 
-### Based on OpenAPI spec 0.0.0
+### Based on SDK spec version 0.0.0 (using OpenAPI 0.0.0)
```

Comment:
It is not really `SDK spec` but on `OpenAPI specification`.
The Specification version with not always aligned with the version of the SDK

Reply:
The goal was to clarify the versions being used as a follow up to this comment: https://github.com/AmadeusITGroup/otter/pull/2267#discussion_r1795601865 
Currently we get the impression that we are specifying the OpenAPI version which is not the case. 
Do you have a suggestion on how to make this heading more clear?

Reply:
Maybe you can write something like:

```javascript
`Based on specifification version ${specVersion[1] || '0.0.0'}

> The specification is following the standard ${generatorVersion[1]} ${generatorVersion[2]}`
```
(or with a better English :D)

What do you think?

Reply:
Good idea, it's more clear that way.
@cpaulve-1A what do you think? (since we implemented this fix together)

Reply:
We just need to ensure it will work with both on legacy and new message (careful, one is single line and the other one is multiline)